apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -18,16 +19,33 @@ const Header: React.FC = () => {
 			<div className={styles["toolbar"]}>
 				<SearchBar />
 
-				<Link to={AppRoute.PROFILE}>
-					<Image
-						alt="user-avatar"
-						className={styles["image"]}
-						height="48"
-						shape="circle"
-						src={user.avatarUrl ?? DEFAULT_USER_AVATAR}
-						width="48"
-					/>
-				</Link>
+				<nav>
+					<ul className={styles["navbar"]}>
+						<li>
+							<Button
+								className={getValidClassNames(
+									styles["navbar-button"],
+									user.hasUnreadNotifications ? styles["unread"] : "",
```

Comment:
As I remember it should give the same result
```suggestion
									user.hasUnreadNotifications && styles["unread"],
```